foia_hub/templates/learn.html

```diff
@@ -25,6 +25,8 @@
         </ul>
       </nav>
 
+        <h2>Didn't find the information you're looking for? Check <a href="FOIA.gov/about.html">FOIA.gov/Learn</a></h2>
```

Comment:
OK, so this is completely crazy but that URL should be all lowercase. It's foia.gov, not FOIA.gov when dealing with the URL. URLs are case sensitive: http://www.w3.org/TR/WD-html40-970708/htmlweb.html